src/lib/exec-fetcher.js

```diff
@@ -1,13 +1,15 @@
 // @ts-check
 const path = require('path')
 const process = require('process')
+const { stringify } = require('querystring')
```

Comment:
[sand] What are your thoughts on using `new URL()` and `url.searchParams.set()` instead?
This is mostly equivalent, except it uses the DOM API instead of Node.js core API. What are your thoughts on this?

Reply:
Yea why not I don't have a preference to be honest -> is there any benefit?

Reply:
If we had more code to share with the front-end, it would be better. And Node.js has been moving in that direction lately of using more and more the DOM API, as opposed to to its own core Node.js modules (`URI`, `AbortSignal`, `EventTarget`, `globalThis`, ES modules, `Blob`, `ReadableStream`/`WritableStream`/`TransformStream`, `Worker`, Web Crypto, `performance`, `structuredClone`), which seems a good thing at a higher level, and is also closer to the direction Deno is heading.

However, this is more of a conceptual benefit in this case, since we do not intend to run this file anywhere but on a terminal in this case, so either way could work.

Reply:
I just realized this too: `querystring` is marked as "legacy" [in the Node.js documentation](https://nodejs.org/api/querystring.html#query-string).

Reply:
I'll change it for you ;)